### DISCUSSION TITLE
Policy.xml of imagemagick???

### DISCUSSION THREAD

**Discussion by n1smo** [beyondcode/herd-community#442] [Q&A] [answered]
This might be a completely dumb questions, but where can i find the policy.xml or imagick files in general that come with herd??
Cant find anything...

**Comment by n1smo** [comment, marked as answer]
For anyone interested in this, it can't be changed with herd.

**Reply by crazywhalecc** [reply]
Herd is using statically linked php binary with various extensions (including imagick), that means it just uses imagemagick as a library (the imagemagick config and binary tools are not included). You can only use API/ABI with PHP funcion.

We used to install imagick extension and dynamic link with imagemagick lib, these fully-installed version of imagemagick will also contain imagick config and binary tools.
